test/benchmark_test.cc

```diff
@@ -55,7 +53,7 @@ static void BM_Factorial(benchmark::State& state) {
   while (state.KeepRunning())
     fac_42 = Factorial(8);
   // Prevent compiler optimizations
-  EXPECT_NE(fac_42, std::numeric_limits<int>::max());
+  std::cout << fac_42;
```

Comment:
while you're here, we should do something better ... maybe use setlabel instead of cout?